### PR TITLE
Translate "Ruby 3.4.0-preview1 Released" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Translation of https://github.com/ruby/www.ruby-lang.org/pull/3240

Actual diff f7d36328e

### CODE REVIEW

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+date: 2024-05-16 00:00:00 +0000
+lang: ko
+---
+
+{% assign release = site.data.releases | where: "version", "3.4.0-preview1" | first %}
+Ruby {{ release.version }} 릴리스를 알리게 되어 기쁩니다.
+
+## 언어 변경
+
+* 파일에 `frozen_string_literal` 코멘트가 없다면, 리터럴 문자열은 이제 얼린 것처럼
```
Comment (yous) [inline ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:15]:
```suggestion
* 파일에 `frozen_string_literal` 주석이 없는 경우, 문자열 리터럴은 이제 얼린 것처럼
```

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  동작합니다. 해당 문자열이 변경되는 경우에도 폐기 예정 경고는 생략됩니다.
```
Comment (yous) [inline ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:16]:
```suggestion
  동작합니다. 해당 문자열이 변경되는 경우에는 폐기 예정 경고가 발생합니다.
```

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  이 경고는 `-W:deprecated`나 `Warning[:deprecated] = true` 설정을 통해 출력할 수 있습니다.
```
Comment (yous) [inline ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:17]:
```suggestion
  이 경고는 `-W:deprecated`나 `Warning[:deprecated] = true` 설정을 통해 활성화할 수 있습니다.
```

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  이 변경을 무효화하고 싶다면 Ruby를 실행할 때 `--disable-frozen-string-literal` 커맨드라인 인수를
+  사용하세요. [[Feature #20205]]
+
+* 블록 인자를 가리키는 `it`이 추가됩니다. [[Feature #18980]]
+
+* 메서드 호출시에 `nil`에 키워드 스플랫을 지원합니다.
```
Comment (yous) [inline ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:23]:
```suggestion
* 메서드 호출 시에 `nil`에 키워드 스플랫을 지원합니다.
```

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+* Exception
+
+  * Exception#set_backtrace는 이제 `Thread::Backtrace::Location`의 배열을 입력으로 받을 수 있습니다.
+    `Kernel#raise`, `Thread#raise`와 `Fiber#raise`도 같은 형식의 입력을 받습니다. [[Feature #13557]]
+
+
+* Range
+
+  * Range#size는 이제 범위가 열거 가능하지 않다면 TypeError를 던집니다. [[Misc #18984]]
```
Comment (yous) [inline ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:43]:
```suggestion
  * Range#size는 이제 범위가 순회 가능하지 않다면 TypeError를 던집니다. [[Misc #18984]]
```

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+## 호환성 문제
+
+주의: 기능 버그 수정은 포함되어 있지 않습니다.
+
+* 에러 메시지와 백트레이스의 출력 결과가 변경됩니다.
+  * 인용 시작 부분을 나타내던 백틱 대신 작은 따옴표를 사용합니다. [[Feature #16495]]
+  * 메서드 이름 앞에 클래스 이름을 출력합니다(클래스가 불변하는 이름을 가지고 있다면). [[Feature #19117]]
+  * `Kernel#caller`, `Thread::Backtrace::Location`의 메서드들도 마찬가지로 변경됩니다.
+  ```
```
Comment (yous) [inline ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:55]: - #3249

````suggestion

  ```
````

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+
+## 호환성 문제
+
+주의: 기능 버그 수정은 포함되어 있지 않습니다.
+
+* 에러 메시지와 백트레이스의 출력 결과가 변경됩니다.
+  * 인용 시작 부분을 나타내던 백틱 대신 작은 따옴표를 사용합니다. [[Feature #16495]]
+  * 메서드 이름 앞에 클래스 이름을 출력합니다(클래스가 불변하는 이름을 가지고 있다면). [[Feature #19117]]
+  * `Kernel#caller`, `Thread::Backtrace::Location`의 메서드들도 마찬가지로 변경됩니다.
```
Comment (yous) [inline ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:54]:
```suggestion
  * `Kernel#caller`, `Thread::Backtrace::Location`의 메서드 등도 마찬가지로 변경됩니다.
```

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+
+## 그 이외의 변경
+
+* 상세 모드(`-w`)에서 메서드에 넘긴 블록이 사용되지 않았을 때
+  경고를 출력합니다.
+  [[Feature #15554]]
+
+* `String.freeze`나 `Integer#+`처럼 인터프리터와 JIT이 특별히 최적화하는
+  몇몇 코어 메서드를 재정의하면 성능 클래스 경고
+  (`-W:performance`나 `Warning[:performance] = true`)를 출력합니다.
```
Comment (yous) [inline ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:83]:
```suggestion
  몇몇 코어 메서드를 재정의하면 성능 클래스
  경고(`-W:performance`나 `Warning[:performance] = true`)를 출력합니다.
```

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:16]: oh 🙈 emitted, not omitted

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md (hunk 6 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:55]: 👍

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md]: fixed 077ff0b

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md]: fixed 077ff0b

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md (hunk 3 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md]: fixed 077ff0b

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md (hunk 4 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md]: fixed 077ff0b

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md (hunk 5 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md]: fixed 077ff0b

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md (hunk 6 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md:56]: fixed 077ff0b

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md (hunk 7 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md]: fixed 077ff0b

File: ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md (hunk 8 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-ruby-3-4-0-preview1-released.md]: fixed 077ff0b

**Comment by yous** [comment]
Thanks!

**Comment by riseshia** [comment]
Thanks for review~~!!

**Pull request merged**